packages/caisar/caisar.0.2.1/opam

```diff
@@ -0,0 +1,65 @@
+opam-version: "2.0"
+synopsis:
+  "A platform for characterizing the safety and robustness of artificial intelligence based software"
+maintainer: [
+  "LAISER team, Software Safety and Security Laboratory, CEA-List"
+]
+authors: ["LAISER team, Software Safety and Security Laboratory, CEA-List"]
+license: "LGPL-2.1-only"
+homepage: "https://git.frama-c.com/pub/caisar"
+doc: "https://git.frama-c.com/pub/caisar"
+bug-reports: "https://git.frama-c.com/pub/caisar/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml" {>= "4.13"}
+  "dune-site" {>= "2.9.0"}
+  "piqi" {>= "0.7.6"}
+  "piqilib" {>= "0.6.14"}
+  "zarith" {>= "1.7"}
+  "ocplib-endian" {>= "1.0"}
+  "base" {>= "v0.14.0"}
```

Comment:
```suggestion
  "base" {>= "v0.16.0"}
```

to fix the following CI error

```
[ERROR] The compilation of caisar.0.2.1 failed at "dune build -p caisar -j 255 --promote-install-files=false @install".

#=== ERROR while compiling caisar.0.2.1 =======================================#
# context              2.2.0~alpha3~dev | linux/x86_64 | ocaml-base-compiler.4.14.1 | pinned(https://git.frama-c.com/api/v4/projects/1082/packages/generic/caisar/0.2.1/caisar-0.2.1.tbz)
# path                 ~/.opam/4.14/.opam-switch/build/caisar.0.2.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p caisar -j 255 --promote-install-files=false @install
# exit-code            1
# env-file             ~/.opam/log/caisar-7-6460cb.env
# output-file          ~/.opam/log/caisar-7-6460cb.out
### output ###
# File "/home/opam/.opam/4.14/lib/biniou/biniou.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.14/lib/ocaml-compiler-libs/bytecomp/ocaml-compiler-libs.bytecomp.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.14/lib/ocaml-compiler-libs/common/ocaml-compiler-libs.common.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.14/lib/ocaml-compiler-libs/shadow/ocaml-compiler-libs.shadow.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.14/lib/ocaml-compiler-libs/toplevel/ocaml-compiler-libs.toplevel.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.14/lib/ppx_derivers/ppx_derivers.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# (cd _build/default && /home/opam/.opam/4.14/bin/ocamlopt.opt -w -40 -g -I src/.main.eobjs/byte -I src/.main.eobjs/native -I /home/opam/.opam/4.14/lib/astring -I /home/opam/.opam/4.14/lib/base -I /home/opam/.opam/4.14/lib/base/base_internalhash_types -I /home/opam/.opam/4.14/lib/base/caml -I /home/opam/.opam/4.14/lib/base/shadow_stdlib -I /home/opam/.opam/4.14/lib/biniou -I /home/opam/.opam/4.14/lib/bos -I /home/opam/.opam/4.14/lib/cmdliner -I /home/opam/.opam/4.14/lib/csv -I /home/opam/.opam/4.14/lib/ctypes -I /home/opam/.opam/4.14/lib/dune-private-libs/dune-section -I /home/opam/.opam/4.14/lib/dune-site -I /home/opam/.opam/4.14/lib/easy-format -I /home/opam/.opam/4.14/lib/fmt -I /home/opam/.opam/4.14/lib/fpath -I /home/opam/.opam/4.14/lib/integers -I /home/opam/.opam/4.14/lib/jane-street-headers -I /home/opam/.opam/4.14/lib/logs -I /home/opam/.opam/4.14/lib/menhirLib -I /home/opam/.opam/4.14/lib/ocaml-protoc-plugin -I /home/opam/.opam/4.14/lib/ocamlgraph -I /home/opam/.opam/4.14/lib/ocplib-endian -I /home/opam/.opam/4.14/lib/ppx_compare/runtime-lib -I /home/opam/.opam/4.14/lib/ppx_deriving/runtime -I /home/opam/.opam/4.14/lib/ppx_deriving_yojson/runtime -I /home/opam/.opam/4.14/lib/ppx_enumerate/runtime-lib -I /home/opam/.opam/4.14/lib/ppx_hash/runtime-lib -I /home/opam/.opam/4.14/lib/ppx_inline_test/config -I /home/opam/.opam/4.14/lib/ppx_inline_test/runtime-lib -I /home/opam/.opam/4.14/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.14/lib/re -I /home/opam/.opam/4.14/lib/result -I /home/opam/.opam/4.14/lib/rresult -I /home/opam/.opam/4.14/lib/seq -I /home/opam/.opam/4.14/lib/sexplib0 -I /home/opam/.opam/4.14/lib/stdio -I /home/opam/.opam/4.14/lib/stdlib-shims -I /home/opam/.opam/4.14/lib/time_now -I /home/opam/.opam/4.14/lib/why3 -I /home/opam/.opam/4.14/lib/yaml -I /home/opam/.opam/4.14/lib/yaml/bindings -I /home/opam/.opam/4.14/lib/yaml/bindings/types -I /home/opam/.opam/4.14/lib/yaml/c -I /home/opam/.opam/4.14/lib/yaml/ffi -I /home/opam/.opam/4.14/lib/yaml/types -I /home/opam/.opam/4.14/lib/yaml/unix -I /home/opam/.opam/4.14/lib/yojson -I /home/opam/.opam/4.14/lib/zarith -I lib/ir/.ir.objs/byte -I lib/ir/.ir.objs/native -I lib/nnet/.nnet.objs/byte -I lib/nnet/.nnet.objs/native -I lib/onnx/.onnx.objs/byte -I lib/onnx/.onnx.objs/native -I lib/ovo/.ovo.objs/byte -I lib/ovo/.ovo.objs/native -I lib/xgboost/.caisar_xgboost.objs/byte -I lib/xgboost/.caisar_xgboost.objs/native -intf-suffix .ml -no-alias-deps -open Dune__exe -o src/.main.eobjs/native/dune__exe__Verification.cmx -c -impl src/verification.pp.ml)
# File "src/verification.ml", line 96, characters 6-27:
# 96 |       Option.value_or_thunk
#            ^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound value Option.value_or_thunk
```

Reply:
We pushed to v0.15.1, as this is the latest version available for ocaml 4.13